.github/workflows/hugo.yml

```diff
@@ -16,6 +16,32 @@ jobs:
         uses: peaceiris/actions-hugo@v2
         with:
           version: 0.83.1
+      - name: Build search index
+        run: |
+          LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/arranf/HugoToJSON/releases/latest)
```

Comment:
I think it would be better to throw this whole script in a `.sh` file and just execute it from here.

Reply:
Added shell script for this.